src/plugins/presentation_util/scripts/optimize-tsconfig.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+require('@kbn/optimizer').registerNodeAutoTranspilation();
+
+const { run } = require('@kbn/dev-utils');
+
+const { optimize } = require('./optimize-tsconfig/optimize');
+const { deoptimize } = require('./optimize-tsconfig/deoptimize');
+
+run(
+  ({ log, flags }) => {
+    const { revert } = flags;
+
+    if (revert) {
+      deoptimize()
```

Comment:
You can just `await deoptimize()`

Reply:
I opted to leave the `finally` block to output the finish messages.

Reply:
Well you've got to await, and if there's an error then the log message probably isn't relevant right? A successful revert should t end with an error